beta/google-cloud-run/README.md

```diff
@@ -172,3 +172,19 @@ To finish setting up automated user provisioning, [connect your identity provide
 3. [Test your SCIM bridge deployment](#step-4-test-your-scim-bridge) using your bearer token.
 
 The new version number that you updated to should appear in the health check, the container logs for 1Password SCIM Bridge, and the top left-hand side of the page if signing in to the SCIM bridge at its URL in a web browser. After you sign in to your SCIM bridge, the [Automated User Provisioning page](https://start.1password.com/integrations/provisioning/) in your 1Password account will also update with the latest access time and SCIM bridge version.
+
+
+## How to update the **scimsession** secret
+
+To use a new `scimsession` credentials file for your SCIM bridge, replace the secret in your Cloud Run:
```

Comment:
- should instruct customer to pause provisioning before rotation
- reworded slightly for clarity

```suggestion
To use regenerated credentials in your SCIM bridge deployment, pause provisioning in your identity provider, then update the value of the `scimsession` credentials:
```